compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -19318,8 +19407,16 @@ OMR::Z::TreeEvaluator::vsetelemEvaluator(TR::Node *node, TR::CodeGenerator *cg)
                }
             else if (size == 4)
                {
-               uint32_t value = valueNode->getIntegerNodeValue<uint32_t>();
-               offset = cg->fe()->findOrCreateLiteral(cg->comp(), &value, size);
+               if (valueNode->getOpCode().isFloat())
+                  {
+                  float value = valueNode->getFloat();
```

Comment:
Minor syntactical nitpick: we call it `value` here but `v` above. Its good to stay consistent for easier parsing / readability.